docs/visual/e2e-testing/setup.md

```diff
@@ -477,6 +477,11 @@ static void test() {
 Now you can run your test and [view your results](#view-test-results) in the Visual Testing Dashboard. Be sure to [review and accept](#accept-baseline) your baseline.
 
 
+## Setting Timeouts
+
+If you're using several visual assertions (such as [snapshots](/visual/e2e-testing/commands-options/#snapshot-command)) in your WebdriverIO test, you may need to increase the `timeout` value in your configuration to help with test stability.
```

Comment:
This link won't work `/visual/e2e-testing/commands-options/#snapshot-command` I think you need to remove the last backslash `/visual/e2e-testing/commands-options#snapshot-command`

Reply:
Hi @luisVelasquezSauce -- Thanks for your review! Our doc authoring tool actually requires the trailing slash before the anchor link. I have tested this link and it does work, taking you directly to the "Snapshot Command" heading on that page, so I am going to resolve these comments.